src/OrchardCore.Modules/OrchardCore.Media/Shortcodes/ShortcodeProperty.cs

```diff
@@ -0,0 +1,54 @@
+﻿namespace OrchardCore.Media.Shortcodes
+{
+    public class ShortcodeProperty
```

Comment:
Mapping short-code properties to the render target (html) seems like it could be much better, in terms of abstraction/reusability, but it was the simplest thing today.